docs/uptime-guide/getting-started.asciidoc

```diff
@@ -0,0 +1,70 @@
+[[uptime-getting-started]]
+== Getting started
+
+The easiest way to get started with Elastic Uptime is by using our
+https://www.elastic.co/cloud/elasticsearch-service[hosted {es} Service] on
+Elastic Cloud. The {es} Service is available on both AWS and GCP,
+and automatically configures {es} and {kib}.
+
+[float]
+=== Hosted Elasticsearch Service
+
+Skip managing your own {es}, and {kib} instance by using our
+https://www.elastic.co/cloud/elasticsearch-service[hosted {es} Service] on
+Elastic Cloud.
+
+https://www.elastic.co/cloud/elasticsearch-service/signup[Try out the {es} Service for free],
+then jump straight to <<install-heartbeat>>.
+
+[float]
+[[before-installation]]
+=== Install the stack yourself
+
+If you'd rather install the stack yourself,
+first see the https://www.elastic.co/support/matrix[Elastic Support Matrix] for information about supported operating systems and product compatibility. Then, follow the steps below.
+
+* <<install-elasticsearch>>
+* <<install-kibana>>
+* <<install-heartbeat>>
+
+[[install-elasticsearch]]
+=== Step 1: Install Elasticsearch
+
+Install an Elasticsearch cluster, start it up, and make sure it's running.
+
+. Verify that your system meets the
+https://www.elastic.co/support/matrix#matrix_jvm[minimum JVM requirements] for {es}.
+. {stack-gs}/get-started-elastic-stack.html#install-elasticsearch[Install Elasticsearch].
+. {stack-gs}/get-started-elastic-stack.html#_make_sure_elasticsearch_is_up_and_running[Make sure elasticsearch is up and running].
+
+[[install-kibana]]
+=== Step 2: Install Kibana
+
+Install Kibana, start it up, and open up the web interface:
+
+. {stack-gs}/get-started-elastic-stack.html#install-kibana[Install Kibana].
+. {stack-gs}/get-started-elastic-stack.html#_launch_the_kibana_web_interface[Launch the Kibana Web Interface].
+
+[[install-heartbeat]]
+=== Step 3: Install and configure Heartbeat
+
+Uptime requires the setup of monitors in Heartbeat.
+These monitors provide the data you'll be visualizing in the {kibana-ref}/xpack-uptime.html[Uptime UI].
+
+See the *Setup Instructions* in Kibana for instructions on installing and configuring Heartbeat.
+Additional information is available in {heartbeat-ref}/heartbeat-configuration.html[Configure Heartbeat].
+
+[role="screenshot"]
+image::images/uptime-setup.png[Installation instructions on the Uptime page in Kibana]
+
+[float]
+==== Important considerations
+
+* Make sure you're using the same major versions of Heartbeat and Kibana.
+
+* Index patterns tell Kibana which Elasticsearch indices you want to explore.
+The Uptime UI requires a `heartbeat-7*` index pattern.
```

Comment:
I'm wondering if there's a way to not hardcode the `7` here. It's not a problem, but it'd be nice to see this number change with majors correctly.

Reply:
Good idea!